backends/azure_table.go

```diff
@@ -91,7 +91,7 @@ func formattedRequestTime() string {
 func (c *AzureTableBackend) Send(ctx context.Context, req *fasthttp.Request, resp *fasthttp.Response, resourceType string, resourceId string) error {
 	date := formattedRequestTime()
 	req.Header.Add("x-ms-date", date)
-	req.Header.Add("x-ms-version", "2017-01-19")
+	req.Header.Add("x-ms-version", "2018-12-31")
```

Comment:
Is this experimental, or is the 2017-01-19 api standard no longer valid?

Reply:
I is still valid, but without the update we'd get an error when calling `Put()`